src/components/DocumentationTopic/DocumentationNav.vue

```diff
@@ -64,6 +64,10 @@
           :objcPath="objcPath"
           :swiftPath="swiftPath"
         />
+        <VersionToggle
+          v-if="versionList"
```

Comment:
because the default value of versionList should be an empty array, then to check that is not empty

```suggestion
          v-if="versionList.length”
```